recipes/r-latticeextra/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '0.6-28' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-latticeextra
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: latticeExtra_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/latticeExtra_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/latticeExtra/latticeExtra_{{ version }}.tar.gz
+  sha256: 780695323dfadac108fb27000011c734e2927b1e0f069f247d65d27994c67ec2
+
+build:
+  number: 0
+  # r-maps is missing in win64 as well
```

Comment:
If `r-maps` is a dependency of this package, then add it to build and run requirements